src/App.js

```diff
@@ -1,24 +1,56 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, {useState} from 'react';
 import './App.css';
+import FoodBox from './components/FoodBox'
+import 'bulma/css/bulma.css';
+import foodList from './foods.json';
+import FoodForm from './components/FoodForm';
+import SearchBar from './components/SearchBar';
+import TodaysFood from './components/TodaysFoods';
+
+let totalCalories = 0;
 
 function App() {
+  const [foodState, setFoodState] = useState(foodList);
+  const [formState, setFormState] = useState(false);
+  const [foodSearch, setFoodSearch] = useState([])
+  const [todayFoodList, setTodayFoodList] = useState([])
+
+  const handleForm =() => {
+    setFormState(!formState)
+  }
+
+  const addFood = (newFood) => {
+    let foodStateCopy = [...foodState]
+    const added = [...foodStateCopy, newFood]
+    setFoodState(added)
+  }
+
+  const foodFilter = (value) => {
+    const filteredFood = foodState.filter((item) => item.name.toLowerCase().includes(value.toLowerCase()));
+    setFoodSearch(filteredFood);
+  }
+
+  const addFoodToTodaysList = (food) => {
+    const updatedTodayList = [...todayFoodList];
+    updatedTodayList.push(food);
+    food.calories *= food.quantity;
+    totalCalories += food.calories
+    setTodayFoodList(updatedTodayList)
+  }
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='AppDiv'>
+      <button onClick={handleForm} className='addButton'>Add new food!</button>
+      {formState && <FoodForm handleForm={handleForm} addFood={addFood}/>}
+      <SearchBar foodFilter={foodFilter} />
+      <div className='body'>
+        <div className='foodBoxes'>
+          {(foodSearch.length === 0 ? foodState : foodSearch).map((foodObject, index) => (
```

Comment:
You could even consider getting the foods always from the same array. If the search input is "", the .includes in the foodFilter will return true.